packages/vue-flicking/src/Flicking.ts

```diff
@@ -102,43 +145,114 @@ export default class Flicking extends Vue {
     this.$_nativeFlicking.removePlugins(removed.map(index => prevList[index]));
   }
 
-  private $_checkCloneCount() {
+  private $_checkCloneCount(): void {
+    const prevCloneCount = this.$_cloneCount;
     const newCloneCount = this.$_nativeFlicking.getCloneCount();
 
-    if (this.$_cloneCount !== newCloneCount) {
-      this.$_cloneCount = newCloneCount;
+    if (prevCloneCount !== newCloneCount) {
       this.$forceUpdate();
     }
   }
 
-  private $_getPanels() {
-    const lastIndex = this.$_nativeFlicking
-      ? this.$_nativeFlicking.getLastIndex()
+  private $_getPanels(h: CreateElement) {
+    const flicking = this.$_nativeFlicking;
+    const slots = this.$slots.default;
+
+    if (!slots) {
+      return [];
+    }
+
+    const oldCloneCount = this.$_cloneCount;
+    const newCloneCount = flicking
+      ? flicking.getCloneCount()
+      : 0;
+
+    const lastIndex = flicking
+      ? flicking.getLastIndex()
       : this.options.lastIndex || DEFAULT_OPTIONS.lastIndex;
-    const panels = this.$slots.default
-      ? [...this.$slots.default.slice(0, lastIndex + 1), ...this.$_getClonedVNodes()]
-      : undefined;
+
+    let panels: VNode[];
+
+    const visibleIndex = this.$_visibleIndex;
+
+    if (this.options.renderOnlyVisible && this.$_slotDiffer) {
```

Comment:
@WoodNeck What is the case that 'this.$_slotDiffer` is invalid?

Reply:
As $_slotDiffer is created on `mounted` lifecycle, it is `undefined` at the first rendering.